lib/core/src/lib/datatable/components/datatable/datatable.component.html

```diff
@@ -16,24 +16,30 @@
             class="adf-datatable-row"
             role="row">
 
-
             <!-- Drag -->
-            <div *ngIf="enableDragRows" class="adf-datatable-cell-header adf-drag-column">
+            <th *ngIf="enableDragRows"
```

Comment:
This does not look correct, the datatable row is actually above not on this element